### PR TITLE
msgxt copilot handoff sample using northwind sample

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Comment by Pawank-MSFT** [comment]
Executed build pipeline: 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/9186903294

**Review by Pawank-MSFT — APPROVED** [review]
Looks correct, Approving!

**Review by Mohammed-MSFT — APPROVED** [review]

File: samples/msgext-copilot-handoff/ts/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  for (const continuationParameter of Object.values(continuationParameters)) {
+    const conversationReference = (continuationParameter as any)
+      .conversationReference;
+
+      await adapter.continueConversationAsync(
+        (continuationParameter as any).claimsIdentity,
+        conversationReference,
+        (continuationParameter as any).oAuthScope,
+        async (context) => {
+          // MicrosoftAppCredentials.trustServiceUrl(
```
Comment (premkaretimsft) [inline samples/msgext-copilot-handoff/ts/src/index.ts:72]: remove this commented out code

File: samples/msgext-copilot-handoff/ts/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+              attachments: [(continuationParameter as any).cardAttachment],
+            },
+            { type: ActivityTypes.Typing },
+            { type: "delay", value: 2000 },
+            {
+              type: ActivityTypes.Message,
+              text: `Do you need revenue or discounts details about ${(
+                continuationToken as string
+              ).replace("-continuation", "")}?`,
+            },
```
Comment (premkaretimsft) [inline samples/msgext-copilot-handoff/ts/src/index.ts:102]: remove this part. Let's stop the flow after the success card is shown.

File: samples/msgext-copilot-handoff/ts/src/index.ts (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/msgext-copilot-handoff/ts/src/index.ts]: removed

File: samples/msgext-copilot-handoff/ts/src/index.ts (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/msgext-copilot-handoff/ts/src/index.ts]: removed

File: samples/msgext-copilot-handoff/ts/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+              text: `Fetching more details using the continuation token passed: ${continuationToken}`,
+            },
+            { type: ActivityTypes.Typing },
+            { type: "delay", value: 4000 },
+            {
+              type: ActivityTypes.Message,
+              text: `Handoff successful!`,
+              attachments: [(continuationParameter as any).cardAttachment],
+            },
+            { type: ActivityTypes.Typing },
```
Comment (premkaretimsft) [inline samples/msgext-copilot-handoff/ts/src/index.ts:92]: We don't need typing indicator after the card is shown so, remove this part as well

Comment (Prithvi-MSFT) [reply samples/msgext-copilot-handoff/ts/src/index.ts]: removed

**Review by premkaretimsft — APPROVED** [review]

**Pull request merged**